oauth2client/contrib/gce.py

```diff
@@ -21,25 +21,20 @@
 import logging
 import warnings
 
-import httplib2
-from six.moves import http_client
-from six.moves import urllib
 
 from oauth2client._helpers import _from_bytes
 from oauth2client import util
-from oauth2client.client import HttpAccessTokenRefreshError
 from oauth2client.client import AssertionCredentials
+from oauth2client.contrib.metadata import MetadataServer
 
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
 logger = logging.getLogger(__name__)
 
-# URI Template for the endpoint that returns access_tokens.
-_METADATA_ROOT = ('http://metadata.google.internal/computeMetadata/v1/'
-                  'instance/service-accounts/default/')
-META = _METADATA_ROOT + 'token'
-_DEFAULT_EMAIL_METADATA = _METADATA_ROOT + 'email'
+# Backwards Compat
```

Comment:
Nothing. Was a comment as part of the review of #476 that since technically this module field doesn't start with an underscore I shouldn't remove it in a non breaking refactor